src/drive/web/modules/actions/useActions.jsx

```diff
@@ -19,15 +19,21 @@ const withHideSelectionBarDispatch = (action, dispatch) => {
 const useActions = (actionCreators, actionOptions = {}) => {
   const dispatch = useDispatch()
   let actions = []
-  actionCreators.map(createAction => {
-    const enhancedAction = withHideSelectionBarDispatch(
-      createAction(actionOptions),
-      dispatch
-    )
-    const name = enhancedAction.label || enhancedAction.icon
-
-    actions.push({ [name]: enhancedAction })
-  })
+  actionCreators
+    .map(createAction => {
+      const action = createAction(actionOptions)
+
+      // Temporarily disable Download button on iOS Flagship app until
+      // the download feature is fixed on this platform
+      // When fixed on this platform, revert PR #2672
```

Comment:
nit: it's not revert PR, it's revert commit https://github.com/cozy/cozy-drive/pull/2672/commits/1d6666ae0c14348e5630afe0f76373e28ac281e8
```suggestion
      // When fixed on this platform, revert PR #2672
```

Reply:
What about `// When fixed on this platform, revert this commit from PR #2672` ?

I cannot link to the commit because its SHA may change after the rebase/merge.

Reply:
Smart solution! Yes!